.gitlab/quartz-jobs.yml

```diff
@@ -79,5 +79,5 @@ quartz_intel_tpls:
         INDEX_SPEC: [~int64]
         PRECISION_SPEC: [double]
   variables:
-    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 ${INDEX_SPEC} precision=${PRECISION_SPEC} +mpi +openmp +hypre ~superlu-dist +lapack +klu"
+    SPEC: "%${COMPILER_SPEC} cstd=99 cxxstd=14 ${INDEX_SPEC} precision=${PRECISION_SPEC} +mpi +openmp +hypre ~superlu-dist +lapack +klu ^suite-sparse@5.13.0"
```

Comment:
Why is a specific version of `suite-sparse` needed?

Reply:
There is a bug in spack@0.19.1 that causes a very old suite-sparse to be picked up unless we specify a version.